### ISSUE TITLE
On-premise tests fail

### ISSUE THREAD

**Issue by berland** [equinor/stea#50]
The on-prem tests (not run in CI) fails currently with a "500 Internal server error":

```
$ pytest tests/test_stea.py::test_calculate -s -v
[snip]
    def get_project(self, project_id, project_version, config_date):
        url = (
            f"{self.server}/api/v1/Alternative/{project_id}/{project_version}/"
            f"summary?ConfigurationDate={date_string(config_date)}"
        )
        try:
            response = requests.get(url, verify=False)
    
            # pylint: disable=no-member
            if not response.status_code == requests.codes.ok:
>               raise HTTPError(
                    f"Could not GET from: {url}  "
                    f"status: {response.status_code} msg: {response.text}"
                )
E               requests.exceptions.HTTPError: Could not GET from: https://st-w4771.statoil.net/api/v1/Alternative/4872/3/summary?ConfigurationDate=2018-06-26T11:00:00  status: 500 msg: <!DOCTYPE html>
E               <html>
E                   <head>
E                       <title>Runtime Error</title>
[snip]
E                           <span><H1>Server Error in '/' Application.<hr width=100% size=1 color=silver></H1>
E               
E                           <h2> <i>Runtime Error</i> </h2></span>
E                             
E                           <b> Description: </b>An application error occurred on the server. The current custom error settings for this application prevent the details of the application error from being viewed remotely (for security reasons). It could, however, be viewed by browsers running on the local server machine.
E                           <br><br>
E               
E                           <b>Details:</b> To enable the details of this specific error message to be viewable on remote machines, please create a &lt;customErrors&gt; tag within a &quot;web.config&quot; configuration file located in the root directory of the current web application. This &lt;customErrors&gt; tag should then have its &quot;mode&quot; attribute set to &quot;Off&quot;.<br><br>

```

**Comment by berland** [comment]
@bpthrane do you know of any changes on the server side, or how we can get hold of the error message on the server side?

**Comment by berland** [comment]
This has now been solved on the server-side (which again uncovered a bug in test code solved by #53 )
